swatch-contracts/deploy/clowdapp.yaml

```diff
@@ -22,6 +22,8 @@ parameters:
     value: quay.io/cloudservices/swatch-contracts
   - name: IMAGE_TAG
     value: latest
+  - name: SWATCH_INTERNAL_SUBSCRIPTION_ENDPOINT
+    value: http://rhsm-capacity-ingress:8000/api/rhsm-subscriptions/v1
```

Comment:
Since we implemented this in swatch-producer-aws, I realized how we can avoid having yet another template parameter/env var we don't need. I will provide details in feedback for `application.properties`. Please revert all `clowdapp.yaml` changes.